content/en/docs/instrumentation/js/exporters.md

```diff
@@ -14,21 +14,137 @@ can learn how to setup exporters following the
 
 {{% /alert %}}
 
-## OTLP endpoint
+## OTLP
 
-To send trace data to a OTLP endpoint (like the [collector](/docs/collector) or
-Jaeger) you'll want to use an exporter package, such as
-`@opentelemetry/exporter-trace-otlp-proto`:
+### Backend
+
+There are many open source and commercial backends available that support OTLP
+to receive telemetry data. Take a look at this
+[non-exhaustive list](/ecosystem/vendors/), to find a backend that fits your
+needs. In this section you will find instructions to set up the
+[OpenTelemetry Collector](/docs/collector/), [Jaeger](https://jaegertracing.io)
+and [Prometheus](https://prometheus.io) to quickly get started.
+
+#### OpenTelemetry Collector
+
+We recommend that setup an [OpenTelemetry Collector](/docs/collector/) to
+receive your telemetry. To try out and verify your instrumentation quickly, you
+can run the collector in a docker container and with a basic configuration, that
+will write all received telemetry to the console.
+
+In an empty directory, create a file called `collector-config.yaml` with the
+following content:
+
+```yaml
+receivers:
+  otlp:
+    protocols:
+      grpc:
+      http:
+exporters:
+  debug:
+    verbosity: detailed
+service:
+  pipelines:
+    traces:
+      receivers: [otlp]
+      exporters: [debug]
+    metrics:
+      receivers: [otlp]
+      exporters: [debug]
+    logs:
+      receivers: [otlp]
+      exporters: [debug]
+```
+
+Now run the collector in a docker container:
+
+```shell
+docker run --rm -v $(pwd)/collector-config.yaml:/etc/otelcol-contrib/config.yaml otel/opentelemetry-collector-contrib
+```
+
+You can now continue with [installing the OTLP exporter packages](#dependencies)
+. Later you may want to [configure the collector](/docs/collector/configuration)
+to send your telemetry to your observability backend.
+
+#### Jaeger (Traces)
```

Comment:
Similar to the getting started or manual instructions I want to rewrite that page in a way that it is self-enclosed. Here a part for that is having a "backend" (collector/jaeger/prom), ready to use and to try out what is described for the different protocols.
Because of that, my line of thinking (for both Jaeger and Prometheus below), was that as a first step people might like to setup a sample environment to send their traces/metrics to (collector is an option, but they like to see that maybe), also this is for OTLP and not for jaeger exporter (which is deprecated anyhow) and prometheus exporter.

Taking another look at that myself, I think there is an option to change this to something like

- Collector & Backend Setup
  - Collector: instructions for the logging collector, but with receivers for all protocols on this page (otlp, zipkin, prometheus)
  - Jaeger: instructions to setup jaeger to accept OTLP and Zipkin
  - Prometheus: instructions to setup prometheus to accept OTLP and "default" prometheus
- OTLP
- Console
- Jaeger
- Prometheus
- Zipkin
- Other available exporters
- Batching spans

In a second step I want to take the top part on setting up the collector/backend and extract it to a common section for all languages once again

wdyt?

Reply:
[3ed156c](https://github.com/open-telemetry/opentelemetry.io/pull/3355/commits/3ed156cae4746f4c9e2f089f00f4c03708e6842e) changed that, please take another look